lib/Doctrine/Access.php

```diff
@@ -96,10 +96,7 @@ public function __unset($name)
     }
 
     /**
-     * Check if an offset axists
-     *
-     * @param   mixed $offset
-     * @return  boolean Whether or not this object contains $offset
+     * @return bool
```

Comment:
Why are some comments removed while others are improved?

Reply:
just fixed what I noticed at the moment :)